### DISCUSSION TITLE
Migrate the specification into a plain text document

### DISCUSSION THREAD

**Discussion by codesoap** [FiloSottile/age#219] [Spec feedback]
I find it inconvenient, that the specification is a Google Docs document. I would perefer the specification to be a simple text document, version controlled with `git`. Those are the drawback, that I see with Google Docs:

- Proposing changes is less clear and less convenient than a pull request would be.
- I cannot see the "true history" of the document and must rely on the "Changes" section.
- The PRIMARY selection ("middle-mouse-button-clipboard") does not work. This is merely an annoyance, but one I encountered repeatedly.
- Google Docs is slow (yes, this is a real complaint - on OpenBSD Firefox is quite slow on JavaScript heavy webpages).
- I cannot `grep` through the document.
- Following links seems somewhat cumbersome to me.

I would suggest creating a separate repository for the specification, because it should be independent of the Go implementation. In case you want to do this, I have already prepared a repo for you to fork: https://github.com/codesoap/age-spec. The redirect of https://age-encryption.org/v1 would need to be changed afterwards.

**Comment by FiloSottile** [comment]
Yes! This is on the TODO list. Thank you for doing the Markdown-ification :)
